R/loadLm.R

```diff
@@ -361,7 +361,7 @@ resampleCoefficients.lm <- function(fit) {
 #' @family simulateSolute
 simulateSolute.loadLm <- function(load.model, flux.or.conc=c("flux","conc"), newdata, 
                                   method=c("parametric", "non-parametric"), from.interval=c("confidence", "prediction"), rho, ...) {
-  
+  s.hat=NULL
```

Comment:
in this case i think R CMD check was catching a real error. please delete line `s.hat=NULL` line and insert this line between the current lines 406 and 407 (just before `noise <- noise * s.hat/sd(noise)`):
```r
s.hat <- summary(load.model@fit)$sigma # residual standard error
```